src/sam/functions/api/get_awards_nsf/app.rb

```diff
@@ -133,7 +134,7 @@ def _transform_response(response_body:)
         return [] unless response_body.is_a?(Hash)
 
         response_body.fetch('response', {}).fetch('award', []).map do |award|
-          next if award['title'].nil? || award['id'].nil? || award['piLastName'].nil?
+          next if award['title'].nil? || award['id'].nil? # || award['piLastName'].nil?
```

Comment:
@briri what does the "#" do in this case?

Reply:
That is a comment in Ruby. So it comments out the last part there

Reply:
@briri Why comment that part out rather than just removing it?